### DISCUSSION TITLE
How to style the default color in dark mode?

### DISCUSSION THREAD

**Discussion by wsw70** [tailwindlabs/tailwindcss#11924] [Help] [answered]
I successfully configured [dark mode][1] in my Vite/Vue/Tailwind app: when applying the `dark` class, any element that has a `dark:...` class is affected as expected.

This works fine on a per-element basis but I do not know how to set up defaults. Specifically, I would like to define a default dark mode font color (the one I have in light/default mode is black).

How can I do that?

  [1]: https://tailwindcss.com/docs/dark-mode

**Comment by wongjn** [comment, marked as answer]
You could consider setting the `color` on a "high" DOM element node, since `color` automatically inherits:

```html
<body class="text-black dark:text-white">
  <p>Will be black in light mode, white in dark mode.</p>
```

**Reply by wsw70** [reply]
Thank you.  I did that before but it did not work, I now realize that the `dark` class was not set high enough in the DOM. Setting it at `<html>` and using what you suggested works just fine.
